Software_Carbon_Intensity/Appendix_A.md

```diff
@@ -1 +1,101 @@
-Add information here
+
+
+**What is SLI and SLO?**
+Reliability is the concept of a system having the quality of being trustworthy or of performing consistently well. Today most applications have a Service level objective (SLO) to facilitate monitoring. SLOs have typically 3 constraints
+1)	Service level indicator or SLI.
+2)	The target metric or Objective in percentile 
+3)	The observation window
+SLI + Objective +Observation window = SLO
+SLIs have an event, a success criterion, and specify where and how you record success or failure. It is specified as the proportion of events that were good. 
+
+Example: Percentage of Query requests that return an error code other than 5XX or a timeout measured on the Query Gateway Frontend
```

Comment:
Maybe specify a bit more what "Query Gateway Frontend" is so that the context of the example is clearer?